modules/core/src/bitgo.ts

```diff
@@ -514,6 +515,7 @@ export class BitGo {
     this._userAgent = params.userAgent || 'BitGoJS/' + this.version();
     this._promise = Bluebird;
     this._reqId = undefined;
+    this._skipHMACVerification = params.skipHMACVerification || false;
```

Comment:
ok. so any env !== prod, or specifically only those environments you just listed?

Reply:
ACK. will throw in ctor if attempting to set this in a prod environment.
I also think we should keep the check in `verifyResponse` since anybody can still set the private flag at runtime and get around the ctor limitation.